src/Files.Backend/Services/Settings/IFoldersSettingsService.cs

```diff
@@ -194,5 +194,10 @@ public interface IFoldersSettingsService : IBaseSettingsService, INotifyProperty
 		/// Gets or sets a value indicating if double clicking a blank space should go up a directory.
 		/// </summary>
 		bool DoubleClickToGoUp { get; set; }
+
+		/// <summary>
+		/// Gets or sets a value indicating the default option to resolve conflicts.
+		/// </summary>
+		FileNameConflictResolveOptionType DefaultResolveOption { get; set; }
```

Comment:
I think this might be better in preferences settings.

Reply:
Moved.